### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danton267**
Improving readme file with tutorial notes.

Please review the changes I make before mering the pull request of my branch into master.
At the bottom of the file is a line I want you to change, please find it and make the change it tells you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +109,44 @@ py -m pip install --upgrade pip
...
+__pycache__/
+credentials1.py
+models/credentials2.py
+```
+This will remove all folders and their content with names `.env` and `__pycache__`. 
+
+It will remove `credentials1.py` file in the root directory and `credentials2` in the `models` directory - but will keep everything else in there.
+
+
+Reviewer: please remove this line of code before merging the pull request.
```
Comment (LazaSony) [inline README.md:151]: Is this the line that needs to be removed?

**Comment by danton267** [comment]
> Is this the line that needs to be removed? 
yes please

File: README.md (hunk 1 above)
Comment (danton267) [reply README.md:151]: yes

Comment (LazaSony) [reply README.md:151]: ok

**Comment by LazaSony** [comment]
Done!

**Pull request merged**